tests/databases/api_tests.py

```diff
@@ -1454,3 +1454,25 @@ def test_validate_parameters_invalid_host(self, is_hostname_valid):
                 },
             ]
         }
+
+    def test_validate_parameters_with_mysql(self):
```

Comment:
I don't understand this test. If we're passing the correct parameters why do we expect the status code to be 422?

You probably only need to update this test (which should fail otherwise with your PR): https://github.com/apache/superset/pull/14208/files#diff-b8428a369bdee6a99d7d4da28161a32d50477174192f9fade2040a335a4f2e42R1254

Reply:
oh huh, I had it with rv.status 200 and it was passing. But you're right.